internal/proto/controller/api/services/v1/target_service.proto

```diff
@@ -77,7 +77,7 @@ service TargetService {
   // AuthorizeSession creates authorization information from a given Target.
   rpc AuthorizeSession(AuthorizeSessionRequest) returns (AuthorizeSessionResponse) {
     option (google.api.http) = {
-      post: "/v1/targets/{id}:authorize-session"
+      post: "/v1/targets/{id=**}:authorize-session"
```

Comment:
Mind adding a comment here about the use of `**`

Reply:
Done.